pkg/service-manager/entities/entities_test.go

```diff
@@ -0,0 +1,24 @@
+///////////////////////////////////////////////////////////////////////
+// Copyright (c) 2017 VMware, Inc. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+///////////////////////////////////////////////////////////////////////
+
+package entities
+
+import (
+	"testing"
+)
+
+func TestServiceClassEntityToModel(t *testing.T) {
+
```

Comment:
Do we want some TODO's here? That would hopefully allow us to better track that these need to be fleshed out.